src/commands/show_config.rs

```diff
@@ -11,6 +12,6 @@ pub(crate) struct ShowConfigCmd {}
 impl Runnable for ShowConfigCmd {
     fn run(&self) {
         let config = RUSTIC_APP.config();
-        println!("{config:#?}");
+        println!("{}", to_string_pretty(config.as_ref()).unwrap());
```

Comment:
What about something like this? Or did you unwrap intentionally because of `human-panic`?

```suggestion
        let Ok(config) = to_string_pretty(config.as_ref()) else {
            status_err!("An error occured, config cannot be shown.");
            PACE_APP.shutdown(Shutdown::Crash);
        }
        
        println!("{config}");
```

Reply:
Or go via `match`:


```suggestion
        match to_string_pretty(config.as_ref()) {
            Ok(config) => println!("{config}"),
            Err(err) => {
                status_err!("An error occured, config cannot be shown: {err}");
                PACE_APP.shutdown(Shutdown::Crash);
            },
        }
        

```

Reply:
Another route:

```rust
impl Display for RusticConfig {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        let Ok(config) = toml::to_string_pretty(self) else {
            return write!(f, "Error: Could not serialize config to TOML");
        };

        write!(f, "{}", config)
    }
}
```

Reply:
I wrote it on Github as a suggestion, so didn't run any checks or formatting